content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -166,18 +166,14 @@ Use the following commands to install Containerd on your system:
 1. Install containerd:
 
    Visit
-   [Getting started with containerd](https://containerd.io/docs/getting-started/#starting-containerd)
+   [Getting started with containerd](https://github.com/containerd/containerd/blob/main/docs/getting-started.md)
    and follow the instructions there, up to the point where you have a valid
-   configuration file (on Linux: `/etc/containerd/config.toml`).
+   configuration file, config.toml.
+   On Linux, you can find this file under the path `/etc/containerd/config.toml`.
+   On Windows, you ocan find this file under the path `c:\Program File\containerd\config.toml`.
```

Comment:
Thank you for the location! A grammatical nit & would we want to generalize the drive location by adding $Env instead of C: ?

```suggestion
   On Windows, you can find this file under the path `C:\Program Files\containerd\config.toml`.
```